src/transaction.js

```diff
@@ -625,24 +793,40 @@ Transaction.prototype.hashForZcashSignature = function (inIndex, prevOutScript,
     var hashSequence = this.getSequenceHash(hashType)
     var hashOutputs = this.getOutputsHash(hashType, inIndex)
     var hashJoinSplits = ZERO
+    var hashShieldedSpends = ZERO
+    var hashShieldedOutputs = ZERO
 
     var bufferWriter
-    var baseBufferSize = 4 * 5 + 32 * 4
+    var baseBufferSize = 0
+    baseBufferSize += 4 * 5
+    baseBufferSize += 32 * 4
     if (inIndex !== VALUE_UINT64_MAX) {
```

Comment:
readability: `VALUE_UINT64_MAX` is a magic number? Please explain why this is so.

Reply:
Done (added a comment in the variable definition).